libs/contrib/Language/JSON.idr

```diff
@@ -6,6 +6,8 @@ import Language.JSON.Parser
 
 import public Language.JSON.Data
 
+import public Text.Bounded
```

Comment:
This module does not live in prelude, base, or contrib, so it will break the use of contrib.'s Language.JSON -- curiously, this does not cause contrib to fail to build, but that must be a side effect of it being built at the same time as the compiler and somehow the compiler's access to a `Text.Bounded` module leaking into the contrib compilation.

Reply:
Correction: You did copy this module into contrib, but the public import will fail until it is added to the exposed modules in the contrib.ipkg file (I believe this is what is going on with the failing Frex compatibility test).

Reply:
Good catch! I added the modules to contrib.ipkg as they appear in idris2api.ipkg. With the exception of `Text.Distance.Levenshtein`, as this would also require pulling `Data.IOMatrix` into contrib. Can also do that of course, but maybe at a later stage if someone needs it.